zowe-rest-api-sample-spring/docs/zos-security.md

```diff
@@ -199,3 +279,45 @@ The shared library `libsecur.so` is expected to be in zFS filesystem in the a di
 #### Links
 
 - <https://www.ibm.com/support/knowledgecenter/en/SSLTBW_2.3.0/com.ibm.zos.v2r3.bpxb100/tls.htm>
+
+## Authorization Checks
+
+The REST API service that is running on z/OS typically needs to check if the user that is authenticated has access to specific SAF resources to protect access to the functionality of the REST API or validate that the user can do the action before the action is started.
+
+The API to check access to SAF resource is a part of `PlatformSecurityService` interface.
+
+It provides following methods:
+
+- `boolean checkPermission(String userid, String resourceClass, String resourceName, AccessLevel accessLevel, boolean resourceHasToExist)`
+- `boolean checkPermission(String userid, String resourceClass, String resourceName, AccessLevel accessLevel)`
+- `boolean checkPermission(String resourceClass, String resourceName, AccessLevel accessLevel, boolean resourceHasToExist)`
+- `boolean checkPermission(String resourceClass, String resourceName, AccessLevel accessLevel)`
+
+All of them return `true` if user has access to the resource and `false` if the user has not access to the resource or the resource does not exist. They throw `AccessControlError` in other cases of failures (invalid user ID, invalid resource name, internal SAF error, misconfiguration of the service). The optional parameter `resourceHasToExist` is by default `true` and it causes that non-existing resource means that that no user has access to it. Some applications may want the opposite, protect access only when the security administator has created the resouces, then they can use `false` as the value of `resourceHasToExist` parameter.
```

Comment:
i think there is a typo here in wording `it causes that non-existing resource means that that no user has access to it`

Reply:
There are multiple typos and wrong word order (good in Czech but not in English). Sorry